rllib/core/rl_module/marl_module.py

```diff
@@ -478,9 +408,9 @@ def build(
 
         if module_id:
             return self.module_specs[module_id].build()
-        return self.marl_module_class.from_multi_agent_config(
-            {"modules": self.module_specs}
-        )
+
+        module_config = self.get_marl_config()
+        return self.marl_module_class(module_config)
 
     def add_modules(
```

Comment:
What for do we need this method?
Why isn't `self.module_specs = {}` to begin with? Seems very shallow to me.